### DISCUSSION TITLE
Monaco Editor Example

### DISCUSSION THREAD

**Discussion by meindonut** [zauberzeug/nicegui#2397] [Q&A]
### Question

I would really like to use the Monaco Diff Editor with nicegui. Has someone already used it and has some hints or better even an example how to implement it?

**Comment by falkoschindler** [comment]
Hi @meindonut,

the Monaco code editor has already been discussed:

- https://github.com/zauberzeug/nicegui/discussions/1675
- https://github.com/zauberzeug/nicegui/discussions/1671
- https://github.com/zauberzeug/nicegui/discussions/1124

Especially #1675 shows and integration of the Monaco editor with NiceGUI.
